### PR TITLE
fix(cyclops-ui): path link on module card fixed

### LINKED ISSUE #336: Unable to edit or delete module if you use a bad repo path
**Describe the bug**
Unable to view the details of a module that was created using a bad repo path. You are just given a "Page not found :/" error and no option to edit or delete the module. 

**To Reproduce**
1. Go to 'http://localhost:3000/modules'
2. Click on 'Add module'
3. Under the "Module template" section type in a bad repo URL and/or path
4. Give the module a name and click Save.
5. In the Deploy Modules page click the Details button of the bad module
image

**Expected behavior**
I'm expecting to be able to view the details of all modules, regardless of whether or not they have a bad repo URL, or path. Instead I'm given a simple 'Page not found :/' and no option to do anything else. 

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: macOS (latest)
 - Browser Chrome 
 - Version latest

### PR DESCRIPTION
**Pull request by s-vamshi**
closes #336

## ✅ Checks
- [ ] I have updated the documentation as required
- [x] I have performed a self-review of my code

## ℹ Additional context

This PR fixes incorrect path url on the module card.
Path URL will contain resolvedVersion if it is not empty else it uses version and if both are not null then _main_

### CODE REVIEW

File: cyclops-ui/src/components/pages/Modules/Modules.tsx [hunk 1]
```diff
@@ -84,6 +84,16 @@ const Modules = () => {
     return version + " - " + resolvedVersion.substring(0, 7);
   };
 
+  const getLinkPath = (version: string, resolvedVersion: string) => {
+    if (resolvedVersion !== "") {
+      return resolvedVersion;
+    }
+    else if(version !== ""){
+      return version;
+    }
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/Modules/Modules.tsx:93]:
```suggestion
    if(version !== ""){
      return version;
    }
```

**Comment by s-vamshi** [comment]
@petar-cvit can you please review this PR in your free time?

**Review by petar-cvit — CHANGES_REQUESTED** [review]

**Comment by s-vamshi** [comment]
@petar-cvit made suggested changes and squashed them

**Review by petar-cvit — APPROVED** [review]
Thanks @s-vamshi 🧡

**Pull request merged**